documentation/modules/proc-creating-users-cli.adoc

```diff
@@ -9,7 +11,13 @@ In {ProductName} users can be created using standard command-line tools.
 
 .Prerequisites
 
-* You must have already created an link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#con-address-space-messaging[address space].
+ifdef::Prereqs[]
```

Comment:
This looks fine, although you could simplify the prereq even further by changing it to, "An address space exists."